src/index.html

```diff
@@ -7,6 +7,634 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__wrapper">
+        <div class="header__logo">
+          <a href="#home" class="header__logo-link">
+            <img
+            alt="Logo"
+            class="header__logo-img"
+            src="./images/logo.png">
+          </a>
+        </div>
+        <nav class="header__nav">
+          <ul class="header__list">
+            <li class="header__item">
+              <a href="#apple" class="header__link is-active">Apple</a>
+            </li>
+            <li class="header__item">
+              <a href="#samsung" class="header__link">Samsung</a>
+            </li>
+            <li class="header__item">
+              <a href="#smartphones" class="header__link">Smartphones</a>
+            </li>
+            <li class="header__item">
+              <a
+              href="#laptops&computers"
+              class="header__link"
+              data-qa="nav-hover">Laptops & Computers</a>
```

Comment:
Attributes still not fixed from previous review
<img width="852" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/64750958/0b9800af-9477-49ea-be1f-b93aa10e6b5a">
```suggestion
              <a
                href="#laptops&computers"
                class="header__link"
                data-qa="nav-hover"
              >
                Laptops & Computers
              </a>
```